--- origsrc/twolame-0.3.12/frontend/Makefile.am	2008-01-09 05:07:36.000000000 -0600
+++ src/twolame-0.3.12/frontend/Makefile.am	2008-01-11 07:29:08.015625000 -0600
@@ -7,4 +7,4 @@
 EXTRA_PROGRAMS = twolame
 
 twolame_SOURCES = frontend.c frontend.h audioin_raw.c audioin_sndfile.c
-twolame_LDADD = $(top_srcdir)/libtwolame/libtwolame.la
+twolame_LDADD = $(top_builddir)/libtwolame/libtwolame.la
--- origsrc/twolame-0.3.12/simplefrontend/Makefile.am	2008-01-09 05:06:09.000000000 -0600
+++ src/twolame-0.3.12/simplefrontend/Makefile.am	2008-01-11 07:29:22.015625000 -0600
@@ -1,5 +1,5 @@
 
-AM_CFLAGS = -I../libtwolame/
+AM_CFLAGS = -I$(top_srcdir)/libtwolame/
 
 # This is just sample code - don't install it
 noinst_PROGRAMS = stwolame
@@ -9,5 +9,5 @@
 	audio_wave.h \
 	audio_wave.c
 
-stwolame_LDADD = $(top_srcdir)/libtwolame/libtwolame.la
+stwolame_LDADD = $(top_builddir)/libtwolame/libtwolame.la
 
