--- origsrc/twolame-0.3.12/configure.ac	2008-01-09 08:55:17.000000000 -0600
+++ src/twolame-0.3.12/configure.ac	2009-04-05 16:54:49.359375000 -0500
@@ -94,8 +94,8 @@ PKG_CHECK_MODULES(SNDFILE, sndfile >= 1.
 	) ]
 )
 			
-AC_SUBST( SNDFILE_CFLAGS )
-AC_SUBST( SNDFILE_LIBS )
+AC_SUBST(SNDFILE_CFLAGS)
+AC_SUBST(SNDFILE_LIBS)
 
 
 
@@ -138,7 +138,7 @@ if test "x$HAVE_SNDFILE" = "xyes"; then
 else 
 	AC_MSG_WARN([Not building twolame frontend because libsndfile is missing.])
 fi
-AC_SUBST( TWOLAME_BIN )
+AC_SUBST(TWOLAME_BIN)
 
 
 dnl ############## Final Output
